docs/diagrams/ArchitectureSequenceDiagram.puml

```diff
@@ -6,7 +6,7 @@ Participant Main #grey
 end box
```

Comment:
For this diagram, shouldn't the lifeline for AddPersonCommand end after the cross?

Reply:
![image](https://user-images.githubusercontent.com/69455499/162260409-131ea555-9a15-43b1-9d5b-e7cfb0dffc5c.png)
He did mention this in the DG regarding PlantUML's limitation